conans/paths/package_layouts/package_cache_layout.py

```diff
@@ -153,15 +153,31 @@ def package_remove(self, pref):
         # with self.update_metadata() as metadata:
         #    metadata.clear_package(pref.id)
 
+    def sources_remove(self):
+        src_folder = os.path.join(self._base_folder, SRC_FOLDER)
+        try:
+            rm_conandir(src_folder)  # This will remove the shortened path too if exists
+        except OSError as e:
+            raise ConanException("%s\n\nFolder: %s\n"
+                                 "Couldn't remove folder, might be busy or open\n"
+                                 "Close any app using it, and retry" % (src_folder, str(e)))
+        scm_folder = os.path.join(self._base_folder, SCM_SRC_FOLDER)
+        try:
+            rm_conandir(scm_folder)  # This will remove the shortened path too if exists
+        except OSError as e:
+            raise ConanException("%s\n\nFolder: %s\n"
+                                 "Couldn't remove folder, might be busy or open\n"
+                                 "Close any app using it, and retry" % (scm_folder, str(e)))
```

Comment:
In this message, I'm always missing information about the actual path being removed, the shortened one, which would be the one blocking.  I agree with the changes as they are now, but this is something to improve.

Reply:
But the OSError exception should contain the failing path, and the exception is printed too, right?

Reply:
Uhm, that's probably right, but I cannot test it outside Windows...